Update terminologies in the updated v6 docs - Closes 1658 (#1675)
LiskHQ/lisk-docs: docs/modules/ROOT/pages/build-blockchain/create-blockchain-app.adoc

The files for the different commands can be adjusted and extended as desired, for example, to include new flags and commands.
<10> `test/`: Contains the test files for the unit, functional, and integration tests.

These files create a ready-to-start sidechain client configured for a local devnet, which uses only the default modules of the Lisk SDK.

[#default-modules]
.Default modules
TIP: Lisk provides a range of *default modules* out of the box.
These modules are created automatically, whenever a sidechain client is bootstrapped via Lisk Commander.

> The default modules provide basic sidechain functionality that the most basic sidechain client requires to run.

Author: ```suggestion
The default modules provide basic sidechain functionality required by a sidechain client to work.
```
